src/main/java/basic/Exer18.java

```diff
@@ -0,0 +1,37 @@
+
+package basic;
+
+public class Exer18 {
+
+    public int convertRomanToDecimal(String romanNumber) {
+        String[][] rule = {
+                {"CM", "900"},
+                {"M", "1000"},
+                {"CD", "400"},
+                {"D", "500"},
+                {"XC", "90"},
+                {"C", "100"},
+                {"XL", "40"},
+                {"L", "50"},
+                {"IX", "9"},
+                {"X", "10"},
+                {"IV", "4"},
+                {"V", "5"},
+                {"I", "1"}
+        };
+        int result = 0;
+        int index = 0;
+
+        for (int i = 0; i < rule.length; i++) {
```

Comment:
Dùng foreach thay vì for-index

Reply:
em cần dùng đến chỉ số index nên em dùng for index

Reply:
Đâu có dùng tới đâu?

Reply:
fixed